mira/metamodel/templates.py

```diff
@@ -6,17 +6,22 @@
 __all__ = ["Concept", "Template", "Provenance", "ControlledConversion", "NaturalConversion"]
 
 import json
+import logging
 import sys
 from collections import ChainMap
 from pathlib import Path
-from typing import List, Mapping, Optional, Tuple
+from typing import List, Mapping, Optional, Tuple, Callable
 
 import pydantic
 from pydantic import BaseModel, Field
 
+from mira.dkg.web_client import is_ontological_child
```

Comment:
might be an issue to have this dependency here, maybe better to import inside the function that calls it

Reply:
Resolved in https://github.com/indralab/mira/pull/19/commits/c00a0d1819c2c6ba8f406811f5f30571f40b9d06 and https://github.com/indralab/mira/pull/19/commits/97461b6a7016a1ba524e959fde6f1d9188934b62